packages/E2ETest/package.json

```diff
@@ -8,48 +8,34 @@
     "lint:fix": "rnw-scripts lint:fix",
     "watch": "rnw-scripts watch",
     "windows": "react-native run-windows",
-    "e2e": "react-native run-windows --no-launch --logging && npm run e2etest",
-    "e2ebundle": "npm run bundle && react-native run-windows --no-launch --no-packager --bundle --logging && npm run e2etest",
-    "e2etest": "rimraf reports/* && wdio run ./wdio.conf.js --colors",
-    "testspec": "wdio --spec",
-    "bundle": "rnw-scripts prepareBundle && react-native bundle --platform windows --entry-file app/index.ts --bundle-output windows/ReactUWPTestApp/Bundle/index.windows.bundle --assets-dest windows/ReactUWPTestApp/Bundle",
-    "buildapp": "react-native run-windows --no-launch --no-packager --no-deploy",
-    "deployapp": "react-native run-windows --no-launch --no-packager --no-build",
-    "buildbundleapp": "npm run bundle && react-native run-windows --no-launch --no-packager --no-deploy --bundle",
-    "deploybundleapp": "react-native run-windows --no-launch --no-packager --no-build --bundle"
```

Comment:
Removed a whole bunch of scripts that were just aggregate commands of other scripts. We weren't really using most of these.